docs/docs/overview/performance.md

```diff
@@ -22,13 +22,15 @@ We used the following configurations for the benchmarks.
 
 - Nodes: 3
 - Machines: `n2d-standard-2`
```

Comment:
```suggestion
- Machines: `n2d-standard-4`
```
I presume

Reply:
Thanks catching that. If `n2d-standard-4` is the default for Constellation, I will have to repeat the benchmarks on GKE with that instance type. I used `-2` there.

Reply:
Yeah, the earlier pre-release versions of Constellation had `n2d-standard-2` as default so it was definitely valid in your previous benchmark. 

Unfortunately, we had to kick out support for `n2d-standard-2` (1c2t) prior to release as updates to Linux/CoreOS included mitigations against Retbleed and caused Linux to disabled SMT, which then caused to fail the K8s pre-flight check.

So `n2d-standard-2` is not supported anymore in 2.0.0 and `n2d-standard-4` is the bottom/default, so comparing it with the same instance type on GKE would be the best I guess. 

I just hope it's not too much effort to recreate the benchmarks...